data/dfndata/npc/humanoids_base.dfn

```diff
@@ -552,4 +552,18 @@ GETML=yomotsuwarrior_se
 GETSA=yomotsuwarrior_se
 GETHS=yomotsuwarrior_se
 GETTOL=yomotsuwarrior_se
+}
+
+//fan dancer - Base NPC with shared DFN tags
+[base_fan_dancer]
+{
+GET=base_humanoid
+NAME=#//fan dancer
+ID=0x0f7
+ORIGIN=se
+}
+
+[fandancer]
+{
+GETSE=fandancer_se
```

Comment:
Add one of these for each expansion, so adding NPC with wrong expansion selected doesn't break the server. We can update where these point at when we have definitions for different eras:

GETUO=fandancer_se
GETT2A=fandancer_se
GETUOR=fandancer_se
GETTD=fandancer_se
GETLBR=fandancer_se
GETAOS=fandancer_se
GETSE=fandancer_se
GETML=fandancer_se
GETSA=fandancer_se
GETHS=fandancer_se
GETTOL=fandancer_se